unison-src/transcripts/fix2053.output.md

```diff
@@ -3,12 +3,10 @@
 
   f a ->
     let
-      go i as acc =
-        match List.at i as with
-          None   -> acc
-          Some a ->
-            use Nat +
-            go (i + 1) as (acc :+ f a)
+      use Nat +
+      go i as acc = match List.at i as with
+        None   -> acc
+        Some a -> go (i + 1) as (acc :+ f a)
```

Comment:
Here we see use of soft hang - the first line of the inner function is a "soft hangable" element (a `match`, but also `do` or `cases` would work), so we swing it up right after the `=`.